run_release.py

```diff
@@ -154,22 +172,28 @@
 that may require changes to your code.
 
 
-Build Changes
+Build changes
 =============
 
 
-C API Changes
+C API changes
 =============
 
-New Features
+New features
 ------------
 
+* TODO
 Porting to Python {version}
 ----------------------
 
+* TODO
+
 Deprecated
 ----------
 
+* TODO
+
+.. Add C API deprecations above alphabetically, not here at the end.
 Removed
 -------
```

Comment:
A unique title means that it will have an `#anchor` instead of `#id<000>`
```suggestion
Removed C APIs
--------------
```